src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/SettingElementTests.cs

```diff
@@ -73,6 +73,8 @@ public void NonDefaultValueHasNonNullHashCode()
                     }
                 }
             };
+
+            Assert.NotEqual(new SettingElement().Value.GetHashCode(), Element.Value.GetHashCode());
```

Comment:
Good catch that this test is incomplete. Howver I don't think this is the right change. Hashcodes aren't guaranteed to have any particular form or value, except that it's a stable value for the life of the process. Both of these could produce a valid hashcode of 0.

The best the test could do is verify that GetHashCode() on this element doesn't throw. Or you can delete the test.

Reply:
Thanks for the feedback!

Since the test is already there I went ahead and verified that `GetHashCode` doesn't throw. I also simplified the check slightly; getting the hashcode on the SettingElement includes calling GetHashCode on SettingValueElement (and so forth), so this check tests the whole chain (currently) and is a bit closer to what the test above is doing as well.

If you have any questions, please let me know. Thanks!